README.md

```diff
@@ -1,35 +1,38 @@
 # Desk_Duck
 An ai chatbot to serve as a "rubber duck" for when you're stuck on a project.
 
-Introduction:
-Desk Duck is an ai application to act as a rubber duck when programming. It is designed to help get over coding hurdles by helping you think through problems and get over errors faster. This project will use eye tracking to determine when a user wishes to speak to the "duck" and stt to accept user input and tts to output an answer conversationally.
-
-
-Contributors:
-Timothy Heidcamp
-Ethan Mcmanus
-Duane Anglin
-Daraun Prince
-Matilda Wang
-Shephali Dubey
-
-
-
-Dependencies:
-VS Code
-pip install langchain or conda install langchain -c conda-forge
-pip install openai langchain-openai
-pip install pypdf
-pip install opencv-python
-pip install mediapipe
-API Key - https://platform.openai.com/docs/overview
-ChatGPT account
-
-
-Summary:
-
-
-
-Sources:
+## Summary:
+
+Desk Duck aims to be a an AI Assistant to help with programming. You can ask it about various programming questions. It is designed to help get over coding hurdles by helping you think through problems and get over errors faster.  Desk Duck uses webcam capabilities to track user's eye contact and begins listening upon contact. It is voice operated and outputs audio and to command line. It uses stt (speech to text) to accept user input and tts (text to speech) to output an answer. In it's current iteration it does not yet interact with the code directly. We are aiming to add that in the future.
+
+## Instructions:
+Clone or Download the repo
+Install dependencies using req.txt and mac_req.txt
+Run Main.py 
```

Comment:
We need to instruct people to go to /app/ in their terminal and I would say run `python main.py`